### ISSUE TITLE
3.0 Toolbar

### ISSUE THREAD

**Issue by carljbowman** [LLK/scratch-gui#1399]
## 2.0 Overview 
The Scratch 2.0 Toolbar contains lots of various information, of which I have tried to break up into smaller categories: Nav/Menus, Account, and Project. 

These sections of information change relative to the state of a project: Share/Unshared, My Project/Someone Else's Project, and Logged-in/Logged-out. Below are the outlined examples of where these states intersect and which information is shown related to that state.

### Logged-out > Project
![logged-out - project](https://user-images.githubusercontent.com/3409578/35748195-2632af4a-081b-11e8-8e84-58259e2c1642.png)

### Logged-out > Someone Else's Project
![logged-out - scratcher project](https://user-images.githubusercontent.com/3409578/35748228-3ec1ac64-081b-11e8-8592-e61129a3182b.png)

### Logged-in > My Project > Unshared
![logged-in - my project - unshared](https://user-images.githubusercontent.com/3409578/35748300-911c1206-081b-11e8-9ce1-d2aa702b9819.png)

### Logged-in > Someone Else's Project
![logged-in - scratcher project](https://user-images.githubusercontent.com/3409578/35748324-a3a77f0a-081b-11e8-8148-06d53e58145a.png)

**Comment by carljbowman** [comment]
## 3.0 Overview
The toolbar will remain relatively the same as Addtional focus has gone into making "Share" much more clear, rethinking the display of messages, more visibility to a project creator, and clarifying state differences between My Project/Someone Else's Project and Shared/Unshared. Some elements will be removed such as "Tips", "About", and addtional tools (e.g. Grow) — all of which are being considered as other pieces of work.

**Things to continue to explore**
- Join Scratch  / Sign-in hierarchy
- Project remixing transition (possibly an animation 
- Introducing a share flow
- Indicated a project is shared
- Dropdown menus
- Responsive scaling

## Preview Version
These sketches focus on the future of the toolbar when all the necessary features are in place. Most of these features are large infrastructural changes (e.g. Save / Load) and will take time to develop. Like other previewed elements, tooltips will display on hover. A preview version can be viewed here: https://github.com/LLK/scratch-gui/issues/1105

## Sketches
Language and icons are still being worked on. 

### Logged-Out > Someone Else's Project 
![logged-out - scratcher project - 1280 x 720](https://user-images.githubusercontent.com/3409578/35750800-36aa4208-0824-11e8-813d-9d42c225a6ec.png)

### Logged-in > Someone Else's Project 
![logged-in - scratcher project - 1280 x 720](https://user-images.githubusercontent.com/3409578/35750853-69d8c492-0824-11e8-9e35-3964746c3541.png)

### Logged-Out > Project > Unshared
![logged-out - project - unshared - 1280 x 720](https://user-images.githubusercontent.com/3409578/35750781-260f9182-0824-11e8-84fd-73b97c10dfb3.png)

### Logged-in > My Project > Unshared
![logged-in - my project - unshared - 1280 x 720](https://user-images.githubusercontent.com/3409578/35750839-5967a6a0-0824-11e8-9dbd-7c337f15451b.png)

### Logged-in > My Project > Shared
![logged-in - my project - shared - 1280 x 720](https://user-images.githubusercontent.com/3409578/35750895-889e8b14-0824-11e8-8330-3012337a9da3.png)

**Comment by kyleplo** [comment]
Nice, but here are a few missing features:
#### > All
Add messages icon
#### > Logged-in > My Project > Shared
Add unshare button
Add delete button?
#### > Logged-in > My Project > Unshared
Add delete button?

**Comment by ktrvs** [comment]
>Add messages icon

I think that was left out on purpose, it's not anywhere on the scratch 2.0 gui:
![1](https://user-images.githubusercontent.com/26391179/35756723-3703dd0a-0864-11e8-9cce-c0c4b13dd7a2.png)

Also, your messages don't have much to do with your projects and other scratchers projects, however, my stuff does as it's really a big a directory of all your projects, and projects are what the editor handles.

On that note though, it wouldn't be a bad exactly be a bad addition, I'm sure many users would quite like that idea.

**Comment by hedgehog125** [comment]
Maybe there could be a way of seeing another one of your project without having to go to "my stuff", as this would make switching between projects faster.

**Comment by python-megapixel** [comment]
Will there be a way to unshare an already shared project from within the editor?

**Comment by hedgehog125** [comment]
@python-megapixel I agree, that would be useful as it can be annoying having to search through my stuff to find it.

**Comment by thisandagain** [comment]
@rschamp @carljbowman @kchadha Moving to April for @rschamp to implement the drop-down menus (blocking @kchadha ).
